docs/specs/tripgo.swagger.yaml

```diff
@@ -3850,3 +3853,19 @@ definitions:
       exchange:
         description: Rate to convert cost from user's local currency to dollar (optional).
         type: number
+
+  EnabledMode:
+    description: MaaS bundle enabled mode for this segment
+    properties:
+      mode:
+        type: string
+        description: mode of transport
```

Comment:
@MarianoTucat Out of curiosity: why is this needed? A segment already has a mode. This just doubles up on that, right? Can this ever be different?

Reply:
This is not needed, we can remove it. Just adding it here because we are actually returning it.